app/models/manageiq/providers/redhat/infra_manager.rb

```diff
@@ -91,4 +51,20 @@ def self.catalog_types
   def self.display_name(number = 1)
     n_('Infrastructure Provider (Red Hat)', 'Infrastructure Providers (Red Hat)', number)
   end
+
+  def self.vm_vendor
+    "redhat".freeze
+  end
+
+  def self.host_vendor
+    "redhat".freeze
+  end
```

Comment:
Can these 2 be moved up between description and catalog_types to match the ordering in the ovirt plugin?

Reply:
:+1: done, moved under `self.description`